policy/modules/system/systemd.te

```diff
@@ -1075,6 +1075,8 @@ allow systemd_resolved_t systemd_networkd_runtime_t:dir watch;
 allow systemd_resolved_t system_dbusd_runtime_t:dir read;
 allow systemd_resolved_t system_dbusd_runtime_t:sock_file read;
 
+allow systemd_resolved_t var_run_t:dir read;
```

Comment:
Youre not allowed to reference var_run_t here directly. instead use something like "files_list_runtime()"
probably also ends up wanting to "watch" (that is if your kernel has support for the watch permission)

Reply:
OK, I'll look into that. Thanks!